packages/web/src/app/App.tsx

```diff
@@ -66,6 +83,9 @@ export const AppInner = () => {
           <Route path='/demo/trpc'>
             <DemoTrpcPage />
           </Route>
+          <Route path={PRIVATE_KEY_EXPORTER_SETTINGS_PAGE}>
```

Comment:
Same question, why not with the rest of the routes? If I understood the mock correctly, this new flow is embedded within settings.